lib/api/model/initial_snapshot.g.dart

```diff
@@ -91,19 +91,29 @@ Map<String, dynamic> _$RecentDmConversationToJson(
 UserSettings _$UserSettingsFromJson(Map<String, dynamic> json) => UserSettings(
       twentyFourHourTime: json['twenty_four_hour_time'] as bool,
       displayEmojiReactionUsers: json['display_emoji_reaction_users'] as bool?,
+      emojiset: $enumDecode(_$EmojisetEnumMap, json['emojiset']),
```

Comment:
(It continues to baffle me that json_serializable emits this inefficient implementation for decoding an enum from JSON — your approach with the `Map` is much better. In principle it's a space/time tradeoff… hmm and I guess I can picture a regime where the approach they've chosen is the most efficient one, particularly in the land of Google where the codebase is gigantic and an app may have thousands of these enums… but there should at least be a way to control it and opt into spending some space to avoid spending linear time.

Anyway, that's a subject for another day. Decoding this setting isn't a hot path anyway.)